app/src/main/java/com/duckduckgo/app/browser/BrowserTabViewModel.kt

```diff
@@ -1892,19 +1903,29 @@ class BrowserTabViewModel @Inject constructor(
         filePathCallback: ValueCallback<Array<Uri>>,
         fileChooserParams: FileChooserParams,
     ) {
-        val acceptTypes = fileChooserParams.acceptTypes
-        val acceptsOnlyImage = acceptsOnly("image/", acceptTypes)
-        val cameraHardwareAvailable = cameraHardwareChecker.hasCameraHardware()
-
-        val mimeTypes = convertAcceptTypesToMimeTypes(acceptTypes)
+        val mimeTypes = convertAcceptTypesToMimeTypes(fileChooserParams.acceptTypes)
         val fileChooserRequestedParams = FileChooserRequestedParams(fileChooserParams.mode, mimeTypes)
+        val cameraHardwareAvailable = cameraHardwareChecker.hasCameraHardware()
 
         command.value = when {
-            fileChooserParams.isCaptureEnabled && acceptsOnlyImage && cameraHardwareAvailable ->
-                ShowImageCamera(filePathCallback)
-            (fileChooserParams.acceptTypes.any { it.startsWith("image/") } && cameraHardwareAvailable) ->
-                ShowExistingImageOrCameraChooser(filePathCallback, fileChooserRequestedParams)
-            else -> ShowFileChooser(filePathCallback, fileChooserRequestedParams)
+            fileChooserParams.isCaptureEnabled -> {
+                when {
```

Comment:
Could we have several acceptTypes or it's either image, video, audio or others? i.e. can we have audio AND video?

Reply:
When `capture` is enabled, it has to be used with one of the types. However, this is not enforced. If the website is not correctly using this, we default to choosing a file from device. I noticed Chrome is showing all possible options. We don't have a UI at the moment that could do this, but it's a good point to address in the future.